blazar/plugins/base.py

```diff
@@ -205,6 +206,11 @@ def add_extra_allocation_info(self, resource_allocations):
                     alloc["extras"]["user_name"] = \
                         lease_to_name[alloc["lease_id"]]
 
+    def add_allocation_cleaning_time(self, resource_allocations, cleaning_time):
+        for allocs in resource_allocations.values():
+            for alloc in allocs:
+                alloc["start_date"] = alloc["start_date"] - datetime.timedelta(minutes=cleaning_time)
```

Comment:
to check my understanding:
we add the cleaning time twice here, to shift both the start time and end time later by the amount of cleaning time.

This is because with cleaning time turned on, the "real" start time will be X amount later than the one listed in the DB.

Reply:
Yes, you can't create a lease within `cleaning_time` on either side of this allocation. This does cause the calendar to show a slight overlap in leases.